[indent] Incorrect expected indentation for jsx spread props

**Tell us about your environment**

<!--
    If you are using ESLint v6.5.0 or later, you can run ESLint with the `--env-info` flag and paste the output here.
-->

* **ESLint Version:** v6.6.0
* **Node Version:** v13.1.0
* **npm Version:** v6.12.1

**What parser (default, Babel-ESLint, etc.) are you using?**

`babel-eslint@10.0.3`

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

<!-- Paste your configuration below: -->
```js
{
  "root": true,
  "parser": "babel-eslint",
  "parserOptions": {
    "sourceType": "module",
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "env": {
    "browser": true,
    "node": false
  },
  "rules": {
    "indent": [2, 2, {"SwitchCase": 1}]
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

<!-- Paste the source code below: -->
```jsx
import React from 'React';

export default function Button(props) {
  return (
    <button
      type="button"
      {
        ...props // ESLint is (incorrectly) expecting less indentation here
      }
    />
  );
};
```

<!-- Paste the command you used to run ESLint: -->
```bash
eslint ./index.js
```

**What did you expect to happen?**

No errors

**What actually happened? Please include the actual, raw output from ESLint.**

```
  8:1  error  Expected indentation of 6 spaces but found 8  indent
```

**Are you willing to submit a pull request to fix this bug?**
Sure, if none of the maintainers have time to get to this.

--- a/lib/rules/indent.js
+++ b/lib/rules/indent.js
@@ -1492,6 +1492,17 @@ module.exports = {
                 );
             },
 
+            JSXSpreadAttribute(node) {
+                const openingCurly = sourceCode.getFirstToken(node);
+                const closingCurly = sourceCode.getLastToken(node);
+
+                offsets.setDesiredOffsets(
+                    [openingCurly.range[1], closingCurly.range[0]],
+                    openingCurly,
+                    1
+                );
+            },
+
             "*"(node) {
                 const firstToken = sourceCode.getFirstToken(node);
 

--- a/tests/lib/rules/indent.js
+++ b/tests/lib/rules/indent.js
@@ -4889,6 +4889,18 @@ ruleTester.run("indent", rule, {
                 <a>baz qux</a>.
             </small>
         `,
+        unIndent`
+            <div
+                {...props}
+            />
+        `,
+        unIndent`
+            <div
+                {
+                    ...props
+                }
+            />
+        `,
         {
             code: unIndent`
                 a(b
@@ -9753,6 +9765,36 @@ ruleTester.run("indent", rule, {
                 </div>
             `,
             errors: expectedErrors([3, 8, 6, "Block"])
+        },
+        {
+            code: unIndent`
+                <div
+                {...props}
+                />
+            `,
+            output: unIndent`
+                <div
+                    {...props}
+                />
+            `,
+            errors: expectedErrors([2, 4, 0, "Punctuator"])
+        },
+        {
+            code: unIndent`
+                <div
+                    {
+                      ...props
+                    }
+                />
+            `,
+            output: unIndent`
+                <div
+                    {
+                        ...props
+                    }
+                />
+            `,
+            errors: expectedErrors([3, 8, 6, "Punctuator"])
         },
         {
             code: unIndent`

